antelope_contracts/tests/erc20/erc20_tester.cpp

```diff
@@ -50,14 +50,8 @@ erc20_tester::erc20_tester(std::string native_symbol_str) : native_symbol(symbol
 
     produce_block();
 
-    create_accounts({eos_token_account, evm_account, token_account, faucet_account_name, erc20_account, evmtok_account});
-
-    auto delegated_auth = authority( 1, {},
-                          {
-                            { .permission = {evmtok_account,config::active_name}, .weight = 1}
-                          });
-
-    set_authority( erc20_account, config::active_name,  delegated_auth );
+    create_accounts({eos_token_account, evm_account, token_account, faucet_account_name, erc20_account});
+    create_account(evmin_account, config::system_account_name, false, true);
```

Comment:
Why is this one done separately from the rest?

Reply:
just to include `eosio.code` in the active permission